vs2026

h-vetinari opened the pull request:
VS2026 GA was just [released](https://learn.microsoft.com/en-us/visualstudio/releases/2026/release-notes) a few days ago. It's binary compatible with the VS2015-VS2022 line.

Needs to wait (at least) for https://github.com/actions/runner-images/issues/13291

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.
 * ℹ️ The recipe is not parsable by parser `conda-recipe-manager`. The recipe can only be automatically migrated to the new v1 format if it is parseable by conda-recipe-manager.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/20653218964](https://github.com/conda-forge/conda-forge-webservices/actions/runs/20653218964). Examine the logs at this URL for more detail.</sub>

h-vetinari commented:
So I checked the license thing manually. At this place in the code
https://github.com/conda-forge/vc-feedstock/blob/6f26fcd71db871f354f9ca45f36459ccfa7c67c1/recipe/vc_repack.py#L203-L209
```python
files = [x.attributes["FilePath"].value  for x in payloads if "FilePath" in x.attributes]
pprint.pprint(files)
```
gets us
```
['wixstdba.dll',
 'thm.xml',
 'thm.wxl',
 'logo.png',
 '1028\\thm.wxl',
 '1029\\thm.wxl',
 '1031\\thm.wxl',
 '1036\\thm.wxl',
 '1040\\thm.wxl',
 '1041\\thm.wxl',
 '1042\\thm.wxl',
 '1045\\thm.wxl',
 '1046\\thm.wxl',
 '1049\\thm.wxl',
 '1055\\thm.wxl',
 '2052\\thm.wxl',
 '3082\\thm.wxl',
 'BootstrapperApplicationData.xml',
 'packages\\VC_Runtime_arm64\\VC_Runtime_arm64.msi',
 'packages\\vcRuntimeMinimum_amd64\\vc_runtimeMinimum_x64.msi',
 'packages\\vcRuntimeAdditional_amd64\\vc_runtimeAdditional_x64.msi',
 'packages\\VC_Runtime_arm64\\cab1.cab',
 'packages\\vcRuntimeMinimum_amd64\\cab1.cab',
 'packages\\vcRuntimeAdditional_amd64\\cab1.cab']
```
IOW, the license file isn't where it used to be (or missing due to a bug).

h-vetinari committed:
order platforms more naturally

h-vetinari committed:
add vs2026

h-vetinari committed:
MNT: Re-rendered with conda-smithy 3.54.1 and conda-forge-pinning 2025.12.30.19.55.26

Other tools:
- conda-build 25.11.1
- rattler-build 0.55.0
- rattler-build-conda-compat 1.4.10

h-vetinari committed:
improve license file handling

h-vetinari committed:
improve license file handling

h-vetinari committed:
apply p7zip cap to unix as well

h-vetinari committed:
add vs2026

h-vetinari committed:
MNT: Re-rendered with conda-smithy 3.61.2 and conda-forge-pinning 2026.05.13.20.17.06

Other tools:
- conda-build 26.3.0
- rattler-build 0.64.1
- rattler-build-conda-compat 1.4.14

isuruf commented on recipe/conda_build_config.yaml:
```diff
@@ -1,50 +1,70 @@
  - b929b7fe-5c89-4553-9abe-6324631dcc3a
 sha256:
- - 1DB5C25643A3A4E4C99BFD0D0931A702A49C73DADC4B30672687F32188C1724C
+ - 90E48ADE404E4576D023ABFA374F323555F233982A8805EA9AC63DCA9491A16B
+ - E912E9E653A5295A382BF86521F9B852AEECA823F4A9BF10BB2FBA7DD9445288
  - D62841375B90782B1829483AC75695CCEF680A8F13E7DE569B992EF33C6CD14A
+ - 1DB5C25643A3A4E4C99BFD0D0931A702A49C73DADC4B30672687F32188C1724C
  - 296F96CD102250636BCD23AB6E6CF70935337B1BBB3507FE8521D8D9CFAA932F
 cross_target_platform:
+ - win-64
  - win-arm64
  - win-64
+ - win-arm64
  - win-64
```
I don't understand the shuffle here. The other changes look good

h-vetinari replied:
I decided to order x64 first in https://github.com/conda-forge/vc-feedstock/pull/110/changes/ad172908c8e8f6f8f99fbf778ffd438336efb4ac, which I considered the more natural order, and the ABABA for `cross_target_platform` is kinda more pleasing than BABAA. I can back it out if you prefer. 🤷

h-vetinari replied:
gentle ping

isuruf replied:
Yes, can you back it out? Makes it easier to review

h-vetinari replied:
I had put it into a separate commit so it would be trivial to review... Anyway, I've backed it out
